node-wasm/src/client.rs

```diff
@@ -373,6 +378,7 @@ impl WasmNodeConfig {
             bootnodes: canonical_network_bootnodes(network.into())
                 .map(|addr| addr.to_string())
                 .collect::<Vec<_>>(),
+            syncing_window_secs: None,
```

Comment:
Should we add `custom_` here too?

Reply:
I initially skipped it, as field could just be omitted in js, but I guess it won't hurt for consistency.